api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/ChildDateOfBirthCalculatorTest.java

```diff
@@ -278,10 +283,49 @@ void shouldReturnChildrenTurningOneAndChildTurningFourInPaymentCycle() {
         assertThat(summary).isEqualTo(expectedSummary);
     }
 
+    @ParameterizedTest(name = "Children dobs={0}")
+    @MethodSource("provideArgumentsForChildrenUnderFour")
+    void shouldReturnChildrenAtStartOrPaymentCycle(List<LocalDate> childrenDobs) {
```

Comment:
Typo: should be `...StartOfPaymentCycle` ('of' not 'or'). Same below.

Reply:
Yep, will fix in the next PR.